cfgov/jinja2/v1/_includes/organisms/chart.html

```diff
@@ -12,25 +12,30 @@
 
    ========================================================================== #}
 
-<div class="chart_wrapper block">
+<div class="o-chart block">
     {% if value.title %}
-        <h3>{{ value.title }}</h3>
+    <h3>{{ value.title }}</h3>
     {% endif %}
 
-  <div class="svg_wrapper"
-       id="{{ value.element_id }}">
+  <div class="m-chart-image"
+       data-title="{{ value.title }}"
+       data-chart-type="{{ value.chart_type }}"
+       data-group="{{ value.group }}"
+       data-source="{{ value.data_source }}"
+       data-description="{{ value.description }}"
+       data-report-type="{{ value.report_type }}">
+       {{ value.description }}
   </div>
 
-  <p class="block__sub block__border-top block short-desc chart_footnote">
-    {% if value.data_source %}
-        <strong>Source:</strong> <a class="icon-link icon-link__download" href="{{ parse_links(value.data_source) | safe }}"> <span class="icon-link_text">Download CSV file</span> </a><br>
+  <p class="m-chart-footnote block__sub block__border-top block short-desc">
+      {% if value.data_source %}
+      <strong>Source:</strong> <a class="icon-link icon-link__download" href="https://s3.amazonaws.com/files.consumerfinance.gov/data/{{ value.data_source }}"> <span class="icon-link_text">Download CSV file</span> </a><br>
       {% endif %}
 
-        <strong>Date published:</strong> automate this with JS<br>
+      <strong>Date published:</strong> January 2017<br>
```

Comment:
@richaagarwal Not a bad idea but we get new data every month so manually updating 100+ chart blocks would suck. Having to redeploy to update the date will also suck but at least we only have to edit it in one place.

Regardless, next sprint we'll solve the problem by using JS to update these dates so for now I'll just leave it be.